pipeline/splicing/calcVarPriors.py

```diff
@@ -1798,7 +1774,10 @@ def getPriorProbAfterGreyZoneSNS(variant, boundaries):
                 "inExonicPortionFlag": "N/A",
                 "CIDomainInRegionFlag": "N/A",
                 "isDivisibleFlag": "N/A",
-                "lowMESFlag": "N/A"}
+                "lowMESFlag": "N/A",
+                "varConsequences": ",".join(varCons)}
+
+    assert False, "Should never reach this"
```

Comment:
Why? Any ideas on more information to provide in the case that it does reach this?